apps/metadata-editor/src/app/dashboard/search-filters-summary/search-filters-summary.component.html

```diff
@@ -0,0 +1,12 @@
+<div
+  class="flex flex-row py-3 px-4 gap-4 overflow-hidden grow border-[1px] border-gray-200"
+  *ngIf="searchFilterActive$ | async"
+>
+  <md-editor-search-filters-summary-item
+    *ngFor="let field of searchFields; let i = index"
+    [fieldName]="field"
+  ></md-editor-search-filters-summary-item>
```

Comment:
I would suggest avoiding creating a separate "summary-item" component, and do everything in the search-filters-summary component. Do you think that would be doable?

Reply:
Hum, I've started with one component in the beginning, but afterwards I felt it became more readable, especially for understanding which field is targeted with a separate component.